app/lib/frontend/pub_client.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
```

Comment:
I'm curious why, does this file belong in frontend folder?

Reply:
Any suggestion where it should belong? It will use the APIs served by the frontend, that felt the most relevant here. Eventually this could become a standalone library (there was a chatter on gitter requesting something along the lines).

Reply:
seems fine for now... maybe `shared/`, but's probably equally bad...